src/spatial/components/Map.module.css

```diff
@@ -0,0 +1,21 @@
+.ol_control button {
+  display: block;
+  margin: 8px;
+  padding: 0;
+  font-size: 2.14em !important;
+  color: white;
+  font-size: 1.14em;
+  font-weight: bold;
```

Comment:
```suggestion
```
Normal font weight is enough. Maybe also check if all other CSS properties are necessary.